app/src/main/java/com/zeapo/pwdstore/SearchableRepositoryViewModel.kt

```diff
@@ -443,8 +443,8 @@ open class SearchableRepositoryAdapter<T : RecyclerView.ViewHolder>(
     private val selectedFiles
         get() = requireSelectionTracker().selection.map { File(it) }
 
-    fun getSelectedItems(context: Context): List<PasswordItem> {
-        val root = PasswordRepository.getRepositoryDirectory(context)
+    fun getSelectedItems(): List<PasswordItem> {
+        val root = PasswordRepository.getRepositoryDirectory()
```

Comment:
can't this use the `root` variable above? It's a getter so the value should always stay 'fresh'

Reply:
The `root` variable is in the model, not the adapter, so that doesn't seem to work. But something much better has now become possible ;-)